src/scales/colorScale.ts

```diff
@@ -8,6 +8,8 @@ export module Scales {
     // The maximum number of colors we are getting from CSS stylesheets
     private static _MAXIMUM_COLORS_FROM_CSS = 256;
 
+    private static _plottableColorCache: string[] = null;
```

Comment:
Specifically setting the variable to null is not needed as that is the default
`private static _plottableColorCache: string[];` is the convention.